client/my-sites/checkout/src/components/wp-checkout.tsx

```diff
@@ -90,7 +90,6 @@ import type {
 import type { RemoveProductFromCart, MinimalRequestCartProduct } from '@automattic/shopping-cart';
 import type { CountryListItem } from '@automattic/wpcom-checkout';
 import type { PropsWithChildren, ReactNode } from 'react';
-
```

Comment:
Since I mentioned it for that one spacing removal and you fixed it, I figured I'd mark the other places this space was removed.